<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_null_controllerSetDialog</class>
 <widget class="QDialog" name="dialog_null_controllerSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_controllerSetDialog_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>控制器-设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_controllerSetDialog_shutdown">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>0</y>
     <width>50</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_controllerSetDialog_waterLow">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>水位低</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_controllerSetDialog_sensorLow">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>传感器</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_controllerSetDialog_timeLow">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>150</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>时长</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_controllerSetDialog_timeLow">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>150</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_controllerSetDialog_waterHigh">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>水位高</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_controllerSetDialog_sensorHigh">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>传感器</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_controllerSetDialog_timeHigh">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>270</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>时长</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_controllerSetDialog_timeHigh">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>270</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_controllerSetDialog_sensorLow">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_controllerSetDialog_sensorHigh">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_controllerSetDialog_use">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>410</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>应用</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_controllerSetDialog_sure">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>410</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_controllerSetDialog_cancel">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>410</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
